### ISSUE TITLE
"speeds" has incorrect data type

### ISSUE THREAD

**Issue by millerdev** [obones/openhab-binding-airzone#1]
Error in OpenHAB Thing configuration
Status: `UNINITIALIZED`
```
HANDLER_INITIALIZING_ERROR
java.lang.IllegalStateException: Expected BEGIN_ARRAY but was NUMBER at line 20 column 17 path $.systems[0].data[0].speeds 
```

Expected no error.

Have I configured something wrong, or is this a bug in the binding code?

### Additional information

According to the [AirZone Local API](https://developers.airzonecloud.com/docs/local-api/#post-/hvac) docs, the `speeds` key is an integer, not an array:

| `speeds: integer` | Maximum fan speed available. Read only. |
|---|---|

Local API result:
```sh
curl -X POST -H "Content-Type: application/json" -d '{"systemID":1,"zoneID":1}' http://IP_ADDR:3000/api/v1/hvac
```
```json
{
	"data":	[{
			"systemID":	1,
			"zoneID":	1,
			"name":	"DKN Plus",
			"on":	0,
			"coolsetpoint":	80,
			"coolmaxtemp":	90,
			"coolmintemp":	60,
			"heatsetpoint":	75,
			"heatmaxtemp":	90,
			"heatmintemp":	60,
			"maxTemp":	90,
			"minTemp":	60,
			"setpoint":	75,
			"roomTemp":	72,
			"modes":	[4, 2, 3, 5, 7],
			"mode":	3,
			"speeds":	3,
			"speed":	0,
			"coldStages":	0,
			"coldStage":	0,
			"heatStages":	0,
			"heatStage":	0,
			"humidity":	0,
			"units":	1,
			"errors":	[],
			"air_demand":	0,
			"floor_demand":	0
		}]
}
```

### Versions
OpenHAB: 4.1.0
AirZone Binding: 0.2.2
AirZone model: AZAI6WSPDKC
AirZone API version:
```sh
curl -X POST -H "Content-Type: application/json" -d '{"systemID":0,"zoneid": 0}' http://IP_ADDR:3000/api/v1/version
```
```json
{
	"version":	"1.50"
}
```

**Comment by obones** [comment]
And yet another API that is inconsistent...
Thanks for the report and the PR, I have made a comment in it where I suggest a code change. I'd rather you change your PR instead of me going ahead, so that you are properly credited for this change.

**Comment by obones** [comment]
Fixed by commit 2719e52e9baf73ce968ba366feea8e2d81a5a702
